### ISSUE TITLE
Giada Crushing after setting audio system as Jack and applying

### ISSUE THREAD

**Issue by MattiaDallaCosta** [monocasual/giada#673]
**Environment**
- OS: Arch Linux x86_64 kernel 6.6.7-zen-1-zen
- Giada version: v0.26.0
- Pipewire version: 1:1.0.0-2

**Describe the bug**
After setting the audio system as jack (to use pipewire-jack as a compatibility layer) and pressing the apply button, the application closes instantly and doesn't set the new config (thing that isn't happening when setting the system to alsa or pulseaudio)

**To Reproduce**
1. Go to 'Config -> Audio'
2. Set the system option to 'Jack'
3. Press the 'Apply' button
4. Everything closes

**Comment by gvnnz** [comment]
Hey @MattiaDallaCosta, can you please share the log messages you get during the crash? Here's a quick guide on how to enable logging -> https://github.com/monocasual/giada/discussions/584. Thanks!

**Comment by MattiaDallaCosta** [comment]
Sure @gvnnz.
This is the full output log:
```
[KA] using API PulseAudio
[KA] Opening stream
     Out device: id=129 channelsCount=2 channelsStart=0
     In device: id=129 channelsCount=1 channelsStart=0
     SampleRate=44100
     BufferSize=1024
RtAudio pulse: _NOT_ running realtime scheduling
[KA] Device opened successfully
[mixer::reset] buffers ready - maxFramesInLoop=529200, framesInBuffer=1024
[mixer::enable] enabled
[KA] Start stream - latency = 0
[KM] Device 'Giada MIDI output': 1 MIDI ports found
  0) Midi Through:Midi Through Port-0 14:0
[KM] Device 'Giada MIDI input': 1 MIDI ports found
  0) Midi Through:Midi Through Port-0 14:0
[MidiMapper::init] total midimaps found: 0
[MidiMapper::read] MIDI map read failed!
[LangMapper::init] total langmaps found: 0
[init] Giada 0.26.0
[init] Build date: Sep  4 2023
[init] Release build
[init] Dependencies:
[init]   FLTK - 1.4.0
[init]   RtAudio - 6.0.1
[init]   RtMidi - 5.0.0
[init]   Libsamplerate
[init]   Libsndfile - libsndfile-1.2.2
[init]   JSON for modern C++ - 3.11.2
[init]   JUCE - 7.0.5
[KA] Compiled RtAudio APIs: 4
  ALSA
  JACK
  PulseAudio
  Dummy
[KM] Compiled RtMidi APIs: 2
  ALSA
  JACK
[KA] using API ALSA
[KA] 8 device(s) found with API ALSA
  id=129) Default ALSA Device
      ins=64 outs=64 duplex=64
      isDefaultOut=true isDefaultIn=true
      sampleRates:
	4000 5512 8000 9600 11025 16000 22050 32000 44100 48000 88200 96000 176400 192000 
  id=130) PulseAudio Sound Server
      ins=32 outs=32 duplex=32
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	4000 5512 8000 9600 11025 16000 22050 32000 44100 48000 88200 96000 176400 192000 
  id=131) Loopback (Loopback PCM)
      ins=32 outs=32 duplex=32
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	8000 9600 11025 16000 22050 32000 44100 48000 88200 96000 176400 192000 
  id=132) Loopback (Loopback PCM)
      ins=32 outs=2 duplex=2
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	8000 9600 11025 16000 22050 32000 44100 48000 88200 96000 176400 192000 
  id=133) HDA Intel PCH (HDMI 0)
      ins=0 outs=8 duplex=0
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	32000 44100 48000 88200 96000 176400 192000 
  id=134) HDA Intel PCH (HDMI 1)
      ins=0 outs=8 duplex=0
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	32000 44100 48000 88200 96000 176400 192000 
  id=135) HDA Intel PCH (HDMI 2)
      ins=0 outs=8 duplex=0
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	32000 44100 48000 88200 96000 176400 192000 
  id=136) HDA Intel PCH (HDMI 3)
      ins=0 outs=8 duplex=0
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	32000 44100 48000 88200 96000 176400 192000 
[KA] using API JACK
[KA] 2 device(s) found with API JACK
  id=129) Built-in Audio Analog Stereo
      ins=8 outs=4 duplex=4
      isDefaultOut=true isDefaultIn=true
      sampleRates:
	48000 
  id=130) Firefox
      ins=2 outs=0 duplex=0
      isDefaultOut=false isDefaultIn=false
      sampleRates:
	48000 
[mixer::disable] disabled
[KA] Opening stream
     Out device: id=129 channelsCount=2 channelsStart=0
     In device: id=129 channelsCount=1 channelsStart=0
     SampleRate=44100
     BufferSize=1024
[KA] RtAudio error 7: RtApi::getDeviceInfo: deviceId argument not found.
zsh: segmentation fault (core dumped)  giada
```
Btw good job on the clarity and cleanness of giada's logs, as a software dev myself I've seen some horrendous garbage

**Comment by gvnnz** [comment]
Thanks! We strive to be as clear as possible for our own mental sanity :face_with_spiral_eyes: 

Anyway, the crash should have been fixed in https://github.com/monocasual/giada/commit/0650238a7a21810ed37bcd6266feac4591df5881 and it's included in the latest release 0.26.1. Please let me know if it works for you now.

**Comment by MattiaDallaCosta** [comment]
I just updated the repo and rebuilt and I can confirm that the last version fixes the problem.
